packages/terra-table/src/Table.module.scss

```diff
@@ -20,17 +20,33 @@
     th,
     td {
       border: 0;
-      border-left: var(--terra-table-cell-border, 0);
+      border-left: var(--terra-table-cell-border-left, 0);
       display: table-cell;
       text-align: left;
       vertical-align: top;
 
       &:first-of-type {
-        border-left: var(--terra-table-row-border-vertical, 1px solid #c8cacb);
+        border-left: var(--terra-table-first-cell-border-left, 1px solid #c8cacb);
       }
 
       &:last-of-type {
-        border-right: var(--terra-table-row-border-vertical, 1px solid #c8cacb);
+        border-right: var(--terra-table-last-cell-border-right, 1px solid #c8cacb);
+      }
+    }
+
+    th {
+      border: 0;
+      border-left: var(--terra-table-header-cell-border-left, 0);
```

Comment:
ah, `th` and `td` are now split, I'll insert those into theme variable names instead of `cell`. Thoughts?

```scss
border-left: var(--terra-table-td-border-left, 0);
...
...
...
border-left: var(--terra-table-th-border-left, 0);
```

Reply:
Left a [comment below](https://github.com/cerner/terra-core/pull/2133/files#r244900559), I'd just stick with `cell` for now.